### DISCUSSION TITLE
Locales

### DISCUSSION THREAD

**Discussion by mopsicus** [TriliumNext/Trilium#1771] [Q&A] [answered]
Does Trilium supported locales for UI? Can I make a new locale and switch UI to it?

**Comment by zadam** [comment, marked as answer]
Hi, no, this is not supported. English is the only language/locale available.

**Reply by mopsicus** [reply]
Pity.. Seems, that feature should be in so great tool.

**Reply by mopsicus** [reply]
Hi @zadam!

I've researched Trilium, and want to use it for my personal knowledge base. But there are some little problems: 1) I want translated UI 2) more light and beautiful theme. As for point 2, I can make new css theme, than point 1 is not available.

What do you think if I try to add internalization to Trilium by i18n?

**Reply by zadam** [reply]
At this point I don't plan i18n. It's not a simple feature to develop and perhaps more importantly it has significant maintenance costs.

As for the theming - I do plan theme overhaul for 0.48 release. If you make some custom theme you can share it here with others :)
